assets/js/admin/form-builder.js

```diff
@@ -1362,6 +1366,35 @@
 			});
 		},
 
+		paymentFieldAppendToQuantity: function( id ) {
+			var dragged_el = $('#' + id);
+
```

Comment:
I'm told the WP Coding standards requires spaces only after variable declaration block, aside from commented ones.